api/src/main/java/marquez/db/DatasetDao.java

```diff
@@ -310,6 +312,10 @@ default Dataset upsertDatasetMeta(
         createNamespaceDao()
             .upsertNamespaceRow(
                 UUID.randomUUID(), now, namespaceName.getValue(), DEFAULT_NAMESPACE_OWNER);
+    DatasetSymlinkRow symlinkRow =
+        createDatasetSymlinkDao()
+            .upsertDatasetSymlinkRow(
+                UUID.randomUUID(), datasetName.getValue(), namespaceRow.getUuid(), true, "", now);
```

Comment:
Can we default the symlink type to `UNKNOWN` or `null` if we don't know the symlink type rather than an empty string?

Reply:
`null` is fine